packages/theming/win32-theme/src/createOfficeTheme.ts

```diff
@@ -66,7 +66,6 @@ export function createOfficeTheme(options: ThemeOptions = {}): ThemeReference {
       }
 
       return {
-        colors: createAliasesFromPalette(theme.host.palette, ref.themeName === 'HighContrast'),
```

Comment:
We need this code to run for when a palette is added. Running this code should be conditional on whether a palette is passed in.

Reply:
(One of your past commits looked like it had what I had in mind but for some reason it's showing as the entire line has been deleted)

Reply:
Ah ic, I can revert it to the conditional setting. Thanks!!